cli/tests/include.bats

```diff
@@ -0,0 +1,172 @@
+#! /usr/bin/env bats
```

Comment:
I just realized we have compose.bats. I think since most of our test files mirror our subcommands I should probably fold that into include.bats